packages/react-next/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -1,152 +1,80 @@
 import * as React from 'react';
-import { classNamesFunction, mergeAriaAttributeValues, warnMutuallyExclusive } from '../../Utilities';
+import { compose } from '@fluentui/react-compose';
+import { mergeAriaAttributeValues } from '../../Utilities';
 import { Icon } from '../../Icon';
-import { ICheckboxProps, ICheckboxStyleProps, ICheckboxStyles } from './Checkbox.types';
+import { ICheckboxProps, ICheckboxSlots } from './Checkbox.types';
 import { KeytipData } from '../../KeytipData';
-import { useId, useControllableValue, useMergedRefs } from '@uifabric/react-hooks';
-import { useFocusRects } from 'office-ui-fabric-react';
+import { useCheckbox } from './useCheckbox';
 
-const getClassNames = classNamesFunction<ICheckboxStyleProps, ICheckboxStyles>({
-  useStaticStyles: true,
-});
+const defaultSlots: Omit<ICheckboxSlots, 'root'> = {
+  input: 'input',
+  checkmark: Icon,
```

Comment:
The base component should not have styled slot references; would it be possible to change this to null and let the composed thing choose?

Reply:
yes that's the plan. i am planning to do that when @khmakoto 's state PR is in. we need to move adding this slot and related state out side of Base component